src/Blueprint/Blueprint/Common/BlueprintConstants.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.Management.WebSites.Version2016_09_01.Models;
+
+namespace Microsoft.Azure.Commands.Blueprint.Common
+{
+    public static class BlueprintConstants
+    {
+        public const string AzureBlueprintAppId = "f71766dc-90d9-4b7d-bd9d-4499c4331c3f";
+        public const string OwnerRoleDefinitionId = "/providers/Microsoft.Authorization/roleDefinitions/8e3af657-a8ff-443c-a75c-2fe8c4bcb635";
+        public const string SubscriptionScope = "/subscriptions/{0}";
+        public const string ManagementGroupScope = "/providers/Microsoft.Management/managementGroups/{0}";
+        public const string BlueprintProviderNamespace = "Microsoft.Blueprint";
+        public const string MgAncestorsRequestUrlTemplate = "https://management.azure.com/subscriptions/{0}?api-version=2018-11-01&$expand=ancestors";
+
+        public static class ParameterSetNames
+        {
+            // Parameter Set names
+            public const string SubscriptionScope = "SubscriptionScope";
+            public const string BySubscriptionAndName = "BySubscriptionAndName";
+            public const string BySubscriptionNameAndVersion = "BySubscriptionNameAndVersion";
+            public const string BySubscriptionNameAndLatestPublished = "BySubscriptionNameAndLatestPublished";
+
+            public const string ManagementGroupScope = "ManagementGroupScope";
+            public const string ByManagementGroupAndName = "ByManagementGroupAndName";
+            public const string ByManagementGroupNameAndVersion = "ByManagementGroupNameAndVersion";
+            public const string ByManagementGroupNameAndLatestPublished = "ByManagementGroupNameAndLatestPublished";
+
+            public const string BlueprintAssignmentsBySubscription = "BlueprintAssignmentsBySubscription";
+            public const string BlueprintAssignmentByName = "BlueprintAssignmentByName";
+            public const string CreateBlueprintAssignment = "BlueprintAssignment";
+            public const string DeleteBlueprintAssignmentByName = "DeleteBlueprintAssignmentByName";
+            public const string DeleteBlueprintAssignmentByObject = "DeleteBlueprintAssignmentByObject";
+        }
+
+        public static class ParameterHelpMessages
+        {
+            public const string SubscriptionId = "Subscription Id.";
```

Comment:
>SubscriptionId [](start = 32, length = 14)

I see them being used in the cmdlet attributes. How are they different from the .md files in the /help folder?

Reply:
The tool that produces the .md files look at the cmdlet attribute "HelpMessage" for the explanations of the parameters.

---
In reply to: [257388050](https://github.com/Azure/azure-powershell/pull/8537#discussion_r257388050) [](ancestors = 257388050)